### ISSUE TITLE
Are relative paths fully supported?

### ISSUE THREAD

**Issue by gabm** [conda/conda-build#2329]
We'd like to use relative paths instead of absolute paths in our scripts on windows and linux:

```
conda-build --croot relative/path/to/something RECIPE_FOLDER
conda-build --test relative/path/to/an/artifact.tar.bz2
...
```

I remember that we had several problems with that in the past so we switched to use absolute paths everywhere. By chance we noticed that something changed recently in path handling in CB3. Seems more robust..

Are the above usages fully supported by CB3 and will they be supported in future?

**Comment by msarahan** [comment]
My intention is that they are supported, but I don't know whether there are explicit tests for relative vs. absolute for every argument.  If you have time, I'd appreciate any contribution you can make towards explicit testing.

**Comment by gabm** [comment]
So it's desired behavior, that relative paths could be used as arguments? in this case I'd provide a PR tomorrow with some testcases to protect relevant usage scenarios for us.

As for the mentioned commands above: croot doesn't seem to work with relative paths .. testing works...

**Comment by msarahan** [comment]
Yes, it's absolutely desired behavior.  Test cases would be much
appreciated.

**Comment by gabm** [comment]
I submitted a PR with the tests and fixes in #2335

**Comment by msarahan** [comment]
fixed by #2335 - closing.
